modules/configuring-localnet-switched-topology.adoc

```diff
@@ -6,22 +6,103 @@
 [id="configuration-localnet-switched-topology_{context}"]
 = Configuration for a localnet switched topology
 
+// To accommodate a link to the NMstate Operator, the content in this module
+// is split with tags. The tag includes don't pull in the module header above.
+
+// tag::localnet-intro[]
 The switched (localnet) topology interconnects the workloads through a cluster-wide logical switch to a physical network.
+// end::localnet-intro[]
+
+// tag::localnet-content[]
+You must map the additional network to the OVN bridge to use it as an OVN-Kubernetes additional network. Bridge mappings allow network traffic to reach the physical network. A bridge mapping associates a physical network name, also known as an interface label, to a bridge created with OVS or OVN.
```

Comment:
Is it a specific `additional network` for this? Bc I think it could be:
```suggestion
You must map an additional network to the OVN bridge to use it as an OVN-Kubernetes additional network. Bridge mappings allow network traffic to reach the physical network. A bridge mapping associates a physical network name, also known as an interface label, to a bridge created with OVS or OVN.
```